src/main/java/frc/robot/subsystems/ClimbPivot.java

```diff
@@ -55,7 +74,79 @@ public Command moveClimbMotor(double speed) {
             });
   }
 
+  public Command toggleClimb() {
+    return Commands.either(
+            startEnd(
+                    () -> {
+                      // climb out
+                      nextMoveOut = false;
+                      climbPivotMotorOne.set(CLIMB_OUT_SPEED);
+                    },
+                    () -> {
+                      climbPivotMotorOne.stopMotor();
+                    })
+                .until(() -> isClimbOut),
+            startEnd(
+                    () -> {
+                      // climb in
+                      nextMoveOut = true;
+                      climbPivotMotorOne.set(CLIMB_IN_SPEED);
+                    },
+                    () -> {
+                      climbPivotMotorOne.stopMotor();
```

Comment:
Should we add periodic() here?  Idea being make sure it gets called and checked in the loop?

Reply:
No- `periodic()` is called every single code loop, and we do not want to stop the motor continuously. [`subsystem.startEnd()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Subsystem.html#startEnd(java.lang.Runnable,java.lang.Runnable)) runs the first action when the command starts and runs the section action (in this case, `climbPivotMotorOne.stopMotor()`) when the command finishes (and the timing is determined by the [`command.until()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Command.html#until(java.util.function.BooleanSupplier)) [decorator method](https://en.wikipedia.org/wiki/Decorator_pattern)).